common/app/views/fragments/nav/editionPickerDropdown.scala.html

```diff
@@ -14,32 +26,16 @@
         id="edition-dropdown-menu"
         aria-hidden="true">
 
-        @Edition.all.map { edition =>
-            @if((edition != Edition(request))) {
-
-                <li class="dropdown-menu__item">
-                    <a data-link-name="nav2 : topbar : edition-picker: @edition.id"
-                        href="@LinkTo(s"/preference/edition/${edition.id.toLowerCase}")"
-                        class="dropdown-menu__title">
+        @editionListItem(Edition(request), true)
 
-                        <span class="u-h">switch to the </span>
-                        @edition.displayName
-                    </a>
-                </li>
-            }
-        }
+        @Edition.others(request).map(e => editionListItem(e))
```

Comment:
Can you see if this works:

```
@Edition.others(request).map(_ => editionListItem(_))
```

if so, lets do that instead.

Reply:
Don't think it's working.